utils/utils.js

```diff
@@ -9,3 +9,13 @@ exports.checkUserExists = (username) => {
       }
     });
 };
+
```

Comment:
This function is doing a very similar job to the function above, perhaps we could make one reusable function instead? (👀 There were some extra notes put in the seminar channel on Tuesday 😄 )

Reply:
Done